tests/codebuild/create_tests.sh

```diff
@@ -18,6 +18,7 @@ function run_canary_tests
   yq w -i testfiles/xgboost-mnist-batchtransform.yaml "spec.modelName" "$(get_sagemaker_model_from_k8s_model xgboost-model)"
   run_test testfiles/xgboost-mnist-batchtransform.yaml 
   run_test testfiles/xgboost-hosting-deployment.yaml
+  run_test testfiles/xgboost-mnist-trainingjob-debugger.yaml
```

Comment:
Also need to add the file to `inject_tests` before it can be used (and replace all bucket names and role ARNs with placeholders)